jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -414,42 +464,11 @@ def import_catalog(self, catalog):
         """
         if isinstance(catalog, str):
             self.catalog.import_file(catalog)
-        elif isinstance(catalog, Table):  # includes QTable
+        elif isinstance(catalog, AstropyTable):  # includes QTable
             self.catalog.import_obj(catalog)
         else:  # pragma: no cover
-            raise ValueError("catalog must be a string (file path)")
+            raise ValueError("catalog must be a string (file path) or Table object")
 
     def vue_do_search(self, *args, **kwargs):
         # calls self.search() which handles all of the searching logic
         self.search()
-
-    def clear_table(self, hide_only=True):
```

Comment:
I don't understand why this PR removes a new method that Hatice just added in https://github.com/spacetelescope/jdaviz/pull/3359

Reply:
this logic was moved into the `clear_table_callback` and was necessary now that we have two tables within the same plugin resulting in conflicting definitions of `clear_table`.  I did remove the `hide_only` kwarg (I can try to reimplement that if we think its useful, or maybe we need to deprecate or make that more clear in the changelog), but otherwise the method should still be available and the functionality essentially preserved (see changes in test coverage for the slight changes in behavior and let me know if you don't agree with any).

Reply:
Ah I just made a separate comment about that test. I didn't write this plugin so I don't know/remember the reason of "hide only" but maybe @javerbukh can recall?